subjects/islower/README.md

```diff
@@ -2,7 +2,7 @@
 
 ### Instructions
 
-Write a function that returns `true` if the `string` passed in parameter only contains lowercase characters, and that returns `false` otherwise.
+Write a function that returns 'true' if the 'string' passed as the parameter contains only lowercase characters; returns 'false' otherwise.
```

Comment:
instead of the `;` we prefer the `and that`

Reply:
"and that" doesn't make sense in the english language.   A semicolon is used to join two related independent clauses in place of a comma and a coordinating conjunction.